packages/styles/scss/themes/expressive/components/_file-uploader.scss

```diff
@@ -28,4 +28,7 @@
     height: rem(20px);
     width: rem(20px);
   }
+  .#{$prefix}--tooltip .#{$prefix}--link {
+    font-size: 1rem;
+  }
```

Comment:
We should put this in the tooltip file no? Doesn't seem like it's part of file-uploader

Reply:
you're on to something !